From: overlays-carrier-updater <a6l@localhost>
Subject: [PATCH] FR: Orange World (208-00/01/02) IPV4V6 home + roaming (Hisense A6L)

A6L device patch (device/hisense/a6l/rom/android, 29 Sep 2026): Lineage lists the Orange
France internet (orange) and initial-attach (ia) APNs as IPv6-only with no roaming
protocol. On the A6L port only an IPv4 data call has been proven and 464XLAT is untested;
dual stack lets the network grant v4, v6 or both. Generated by
device/hisense/a6l/rom/android/apn/a6l_apn_fixup.py.
---
--- a/FR.xml
+++ b/FR.xml
@@ -21,7 +21,8 @@
         mnc="00"
         carrier="Orange World"
         apn=""
-        protocol="IPV6"
+        protocol="IPV4V6"
+        roaming_protocol="IPV4V6"
         type="ia"
     />
     <apn
@@ -33,7 +34,8 @@
         bearer_bitmask="1|2|3|9|10|11|13|14|15|20"
         mmsc="http://mms.orange.fr"
         password="orange"
-        protocol="IPV6"
+        protocol="IPV4V6"
+        roaming_protocol="IPV4V6"
         type="default,dun,supl,xcap,mms"
         user="orange"
     />
@@ -305,7 +307,8 @@
         mnc="01"
         carrier="Orange World"
         apn=""
-        protocol="IPV6"
+        protocol="IPV4V6"
+        roaming_protocol="IPV4V6"
         type="ia"
     />
     <apn
@@ -317,7 +320,8 @@
         bearer_bitmask="1|2|3|9|10|11|13|14|15|20"
         mmsc="http://mms.orange.fr"
         password="orange"
-        protocol="IPV6"
+        protocol="IPV4V6"
+        roaming_protocol="IPV4V6"
         type="default,dun,supl,xcap,mms"
         user="orange"
     />
@@ -489,7 +493,8 @@
         mnc="02"
         carrier="Orange World"
         apn=""
-        protocol="IPV6"
+        protocol="IPV4V6"
+        roaming_protocol="IPV4V6"
         type="ia"
     />
     <apn
@@ -501,7 +506,8 @@
         bearer_bitmask="1|2|3|9|10|11|13|14|15|20"
         mmsc="http://mms.orange.fr"
         password="orange"
-        protocol="IPV6"
+        protocol="IPV4V6"
+        roaming_protocol="IPV4V6"
         type="default,dun,supl,xcap,mms"
         user="orange"
     />
